File: CC_Visual Basic/VisualBasic/BeginVBE/BeginVBE/BVBE Projects/Loan/Form1.vb
Public Class Form1

  Private Sub btnCompute_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnCompute.Click
    Dim Loan As Single
    Dim Interest As Single
    Dim Months As Integer
    Dim Payment As Single
    Dim Multiplier As Single
    'Read text boxes
    Loan = Val(txtLoan.Text)
    Interest = Val(txtInterest.Text)
    Months = Val(txtMonths.Text)
    'Compute interest multiplier
    Multiplier = (1 + Interest / 1200) ^ Months
    'Compute payment
    Payment = Loan * Interest * Multiplier / (1200 * (Multiplier - 1))
    txtPayment.Text = "$" & Format(Payment, "0.00")
  End Sub
End Class

File: CC_C#/BVCS Projects/Loan/Form1.cs
#region Using directives

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Windows.Forms;

#endregion

namespace Loan
{
    partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnCompute_Click(object sender, EventArgs e)
        {
            double loan, interest, months, payment, multiplier;
            // Read text boxes
            loan = Convert.ToDouble(txtLoan.Text);
            interest = Convert.ToDouble(txtInterest.Text);
            months = Convert.ToDouble(txtMonths.Text);
            // Compute interest multiplier
            multiplier = Math.Pow((1 + interest / 1200), months);
            // Compute payment
            payment = loan * interest * multiplier / (1200 * (multiplier - 1));
            txtPayment.Text = "$" + Convert.ToString(String.Format("{0:f2}", payment));
        }
    }
}